pubsublite/internal/wire/subscriber.go

```diff
@@ -492,17 +510,22 @@ func (as *assigningSubscriber) handleAssignment(partitions partitionSet) error {
 	return nil
 }
 
-func (as *assigningSubscriber) doHandleAssignment(partitions partitionSet) ([]*singlePartitionSubscriber, error) {
+func (as *assigningSubscriber) doHandleAssignment(nextPartitions PartitionSet) (PartitionSet, []*singlePartitionSubscriber, error) {
```

Comment:
Nit; can you add a comment explaining that this returns the older partition set and the removed subscribers?

Reply:
Done